docs/release-notes/1.3.0/release-1.3.0.md

```diff
@@ -0,0 +1,97 @@
+
+  ![DNN stack](dnn_stack.jpg)
+
+  ```cs
+  public static DnnEstimator RetrainDnnModel(
+            this ModelOperationsCatalog catalog,
+            string[] outputColumnNames,
+            string[] inputColumnNames,
+            string labelColumnName,
+            string tensorFlowLabel,
+            string optimizationOperation,
+            string modelPath,
+            int epoch = 10,
+            int batchSize = 20,
+            string lossOperation = null,
+            string metricOperation = null,
+            string learningRateOperation = null,
+            float learningRate = 0.01f,
+            bool addBatchDimensionInput = false,
+            DnnFramework dnnFramework = DnnFramework.Tensorflow)
+
+  public static DnnEstimator ImageClassification(
+            this ModelOperationsCatalog catalog,
+            string featuresColumnName,
+            string labelColumnName,
+            string outputGraphPath = null,
+            string scoreColumnName = "Score",
+            string predictedLabelColumnName = "PredictedLabel",
+            string checkpointName = "_retrain_checkpoint",
+            Architecture arch = Architecture.InceptionV3,
+            DnnFramework dnnFramework = DnnFramework.Tensorflow,
+            int epoch = 10,
+            int batchSize = 20,
+            float learningRate = 0.01f,
+            bool measureTrainAccuracy = false)
+
+  ```
+
+  [Design specification](https://github.com/dotnet/machinelearning/blob/cd591dd492833964b6829e8bb2411fb81665ac6d/docs/specs/DNN/dnn_api_spec.md)
+
+  [Image classification (Inception V3) sample](https://github.com/dotnet/machinelearning/blob/b6146219156cb05e2e2cebd80fe4cda017f03470/docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/InceptionV3TransferLearning.cs))
+
+  [Image classification (Resnet V2 101) sample](https://github.com/dotnet/machinelearning/blob/b6146219156cb05e2e2cebd80fe4cda017f03470/docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearning.cs)
+
+- **PyTorch scoring support (PREVIEW)** ([#4054](https://github.com/dotnet/machinelearning/pull/4054))  
+  Introduces `Microsoft.ML.PyTorch` package that will enable scoring pytorch models in ML .NET. We plan to bring in PyTorch training next release.
+  ```cs
+  public static TorchModel LoadTorchModel(this ModelOperationsCatalog catalog, string modelLocation)
+  ```
+
+- **SQL Dataloader (PREVIEW)** ([#4035](https://github.com/dotnet/machinelearning/pull/4035))  
```

Comment:
It might be better if we don't call it "SQL Dataloader" but "Database Loader (PREVIEW)" since folks can think it only supports SQL Server if using SQL as part of the name/title. In the release notes we should say that this loader supports any relational database supported by System.Data in .NET Framework or .NET Core, meaning that you can use many RDBMS such as SQL Server, Azure SQL Database, Oracle, PostgreSQL, MySQL, etc. 
Also, the name is DatabaseLoader, we're not really using the acronym SQL as part of the name precisely to avoid confusion identifying it with SQL Server (Since in .NET System.Data.SqlClient targets only SQL Server family).